app/views/feedback/create.html.haml

```diff
@@ -0,0 +1,3 @@
+Thanks for your feedback
+
+= link_to "Return to gov.au", @feedback.url
```

Comment:
In that case, this page has all the same accessibility problems as the `new` page. Also, you probably meant the `Thanks...` and `link_to` to be in separate `<p>` tags so they end up on different lines.